cmake/noa/library.cmake

```
@@ -50,23 +59,34 @@ function(noa_library)
 
   add_library(${ALIAS_NAME} ALIAS ${TARGET_NAME})
 
+  if(NOT NOA_LIBRARY_VARIANT)
+    set(includeDir "${CMAKE_CURRENT_SOURCE_DIR}/include")
```

Please do snake_case on CMake:

```suggestion
    set(include_dir "${CMAKE_CURRENT_SOURCE_DIR}/include")
```